tools/cdi-func-test-file-host-init/nginx.conf

```diff
@@ -120,7 +120,7 @@ http {
             proxy_set_header X-Forwarded-Proto "https";
             proxy_read_timeout                 900;
 
-            limit_rate 100k;
+            limit_rate 1M;
```

Comment:
@awels It seems 100k (per second) was too harsh. The test was taking ages to run.